recipes/full_finetune_distributed.py

```diff
@@ -722,7 +732,7 @@ def train(self) -> None:
                     # Update the number of steps when the weights are updated
                     self.global_step += 1
 
-                    loss_to_log = loss.item()
+                    loss_to_log = running_loss.item() / num_tokens
```

Comment:
Should probably normalize by local_num_tokens?

Reply:
Update: I am probably gonna keep it like this since it should be representative of the loss we are actually using to step (even though it means our loss curves will look slightly different than they do today)

Reply:
i think it makes sense. Will it break all regression tests though?